setup.py

```diff
@@ -17,7 +17,6 @@
         "test": [
             "httpx>=0.18",
             "Pillow>=8.2",
-            "httpx>=0.18",
```

Comment:
why are you touching this file?

Reply:
Boyscout rule ;) I can revert if it's not desired.

Reply:
I'd move it to a separate PR then.

Reply:
Reverted.